package.json

```diff
@@ -38,10 +38,12 @@
     "cryptoassets",
     "ethereum",
     "foundry",
+    "lockup",
     "money-streaming",
     "real-time-finance",
     "sablier",
     "sablier-v2",
```

Comment:
should we remove "sablier-v2"?

Reply:
I left it intentionally. These are search keywords so having "sablier-v2" would result into this repo which is a good thing.